domains/kenny.json

```diff
@@ -1,14 +1,11 @@
 {
     "owner": {
-        "username": "FireBoiDiscord",
+        "username": "KennySB-dev",
         "email": "kennybugeja@gmail.com"
     },
     "record": {
-        "A": [
-            "217.174.245.249",
-            "51.161.54.161"
-            ],
+        "CNAME": "kennysb-dev.github.io",
         "MX": ["mail.is-a.dev"],
         "TXT": "v=spf1 mx a:mail.is-a.dev ~all"
     }
```

Comment:
sorry but you cant use CNAME with MX and TXT

Reply:
wait so how can i use my own site hosted on github, and still use the mail hosted by is-a.dev?

Reply:
you can use A records with github pages, i use github pages with A record (tho you need to look up the A records for your github page)

Reply:
so instead of cname ill use a records

Reply:
mhm, look up the A records for kennysb-dev.github.io then replace CNAME with A records